### DISCUSSION TITLE
RFControl using pilight

### DISCUSSION THREAD

**Discussion by zackdvd** [HomeSpan/HomeSpan#723] [General]
Hi,
is there a simple way to "convert" from pilight to the build in RF sender RFControl.h ?
Pilight documentation gives [examples](https://manual.pilight.org/protocols/433.92/switch/intertechno.html) and of course source code, can it be easily transferred?

Thank you for any kind of input!

**Comment by HomeSpan** [comment]
There is no way to incorporate this into HomeSpan.  However, if pilight has the ability to print out the timing sequence it uses for various remotes, you can use those as input into RMT.
